nav2_system_tests/src/error_codes/test_error_codes_launch.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+# Copyright (c) 2019 Samsung Research America
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch import LaunchService
+from launch.actions import ExecuteProcess, IncludeLaunchDescription
+from launch_testing.legacy import LaunchTestService
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+
+def generate_launch_description():
```

Comment:
Why does this not only launch the planner/controller servers?

Reply:
Same as above

Reply:
Please fix.